spec/features/admin/tasks/further_education_payments/fe_provider_verification_v2_spec.rb

```diff
@@ -106,21 +152,11 @@
         expect(claimant_answer).to have_content("A or AS level physics")
         expect(provider_answer).to have_content("Yes")
       end
-
-      within_table_row("Performance measures") do |claimant_answer, provider_answer|
-        expect(claimant_answer).to have_content("No")
-        expect(provider_answer).to have_content("No")
-      end
-
-      within_table_row("Disciplinary action") do |claimant_answer, provider_answer|
-        expect(claimant_answer).to have_content("No")
-        expect(provider_answer).to have_content("No")
-      end
     end
   end
 
   def within_table_row(label, &block)
-    within(first("tr", text: label)) do
+    within(first("th", exact_text: label).find(:xpath, "./..")) do
```

Comment:
It's fine to change this but I'm just wondering why it needed to change if we've not changed any view files?